apps/aecore/lib/aecore/chain/worker.ex

```diff
@@ -680,25 +649,31 @@ defmodule Aecore.Chain.Worker do
       {key = :accounts, root_hash}, acc_state ->
         Map.put(acc_state, key, PatriciaMerkleTree.new(key, root_hash))
 
-      # TODO
-      # This workaround was made until the Oracles were converted to PatriciaMerkleTree #GH-349
+      {_key = :oracles, %{otree: o_root_hash, ctree: c_root_hash}}, acc_state ->
+        otree = %{:otree => PatriciaMerkleTree.new(:oracles, o_root_hash)}
+        ctree = %{:ctree => PatriciaMerkleTree.new(:oracles_cache, c_root_hash)}
```

Comment:
can we rename `otree`, `o_root_hash` and `ctree`, `c_root_hash` to something nicer?

Reply:
What about `oracle_tree: oracle_root_hash` and `cache_tree: cache_root_hash` ?

Reply:
perfect, maybe even `oracle_cache_...`